template/src/pages/index.tsx

```diff
@@ -126,86 +129,128 @@ export const button = css({
     '&:active': buttonHoverStyle,
 });
 
-const Home: NextPage = () => (
-    <div className={container()}>
-        <Head>
-            <title>Create Next App</title>
-            <meta name="description" content="Generated by create next app" />
-            <link rel="icon" href="/favicon.png" />
-        </Head>
-
-        <main className={main()}>
-            <Heading>
-                This is Ackee <a href="https://nextjs.org">Next.js</a> template!
-            </Heading>
-
-            <p className={description()}>
-                Get started by editing <code className={code()}>pages/index.tsx</code>
-            </p>
-
-            <div className={grid()}>
-                <a href="https://nextjs.org/docs" className={card()}>
-                    <h2>Documentation &rarr;</h2>
-                    <p>Find in-depth information about Next.js features and API.</p>
-                </a>
+export const getServerSideProps: GetServerSideProps = async context => {
+    const queryClient = new QueryClient();
 
-                <a href="https://nextjs.org/learn" className={card()}>
-                    <h2>Learn &rarr;</h2>
-                    <p>Learn about Next.js in an interactive course with quizzes!</p>
-                </a>
+    const testDataQueryFn = async () => getQueryData(config.endpoints.testData);
+
+    // Note: How to fetch data on server side
+    await queryClient.prefetchQuery([QueryKeyName.TEST_DATA], testDataQueryFn);
+
+    return {
+        props: {
+            dehydratedState: dehydrate(queryClient),
+        },
+    };
+};
+
+const Home: NextPage = () => {
+    // Note: How to fetch data on client side
+    const { data, isError, isLoading, isSuccess } = useAppQuery([QueryKeyName.TEST_DATA], config.endpoints.testData);
+
+    const { mutate } = useAppMutation(config.endpoints.testData, {
+        // Note: optional options
+    });
 
-                <div className={card()}>
-                    <a href="https://github.com/AckeeCZ/next-app-template/tree/main/docs/sentry.md">
-                        <h2>How to use Sentry &rarr;</h2>
-                        <p>Find out how to configure Sentry in the app and test the setup with button below.</p>
+    // eslint-disable-next-line no-console
+    console.log(data, isError, isLoading, isSuccess);
```

Comment:
console.log

Reply:
it is here on purpose to see the result from API, but good point and I will add a TODO to remove it